collect-logs

```diff
@@ -331,13 +365,13 @@ def collect_logs(juju):
     """
     units = get_units(juju)
     # include bootstrap
-    units.append("0")
+    units.append(JujuUnit("0", get_bootstrap_ip(juju)))
 
     log.info("Collecting running processes for all units including bootstrap")
     map(partial(_create_ps_output_file, juju), units)
 
     log.info("Creating remote tarball in parallel for units %s" % (
-        ",".join(units)))
+        ",".join([u[0] for u in units])))
```

Comment:
fixed. yeah it can be referenced either way equivalently, but this was an oversight.